examples/api/lib/material/expansion_tile/expansion_tile.1.dart

```diff
@@ -0,0 +1,56 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flutter code sample for ExpansionTile
+
+import 'package:flutter/material.dart';
+
+void main() => runApp(const ExpansionTileApp());
+
+class ExpansionTileApp extends StatelessWidget {
+  const ExpansionTileApp({Key? key}) : super(key: key);
```

Comment:
nit: you'll have to change these to super parameters. That lint is now enforced for doc samples as well.